src/services/signin.ts

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+import { deleteAuthToken } from "utils/deleteAuthToken";
+import { setAuthToken } from "utils/setAuthToken";
+
+export const singin = async (email: string, password: string) => {
+    return axios.post('http://localhost:4200/login', {
```

Comment:
SERVER_URL should be contained in .env, moreover you should create axios's instance with that server url and config and use it here like `api.post('/login', data)`. You can also add transformers (not the movie xdd)